common-lib/src/main/scala/com/gu/mediaservice/lib/cleanup/SupplierProcessors.scala

```diff
@@ -431,19 +431,25 @@ object ReutersParser extends ImageProcessor {
         suppliersReference = extractFixtureID(image) orElse image.metadata.suppliersReference
       )
     )
-    // Others via Reuters
-    case Some("USA TODAY Sports") => image.copy(
+    case Some(credit) if (credit.toLowerCase.contains("reuters") && credit.toLowerCase.contains("agencja")) && (credit.toLowerCase.contains("wyborcza") || (credit.toLowerCase.contains("gazeta"))) => image.copy(
```

Comment:
cleaner brackets...
```suggestion
    case Some(credit) if credit.toLowerCase.contains("reuters") && credit.toLowerCase.contains("agencja") && (credit.toLowerCase.contains("wyborcza") || credit.toLowerCase.contains("gazeta")) => image.copy(
```

Reply:
done when we mobbed, to produce https://github.com/guardian/grid/pull/3668/commits/18955fe1015e68dc2c8da25355f0081a84089ad6